scripts/rename_resources.py

```diff
@@ -54,7 +52,7 @@ def main(argv: list[str]):
                         action='store_true',
                         help='Report status without altering resources')
     args = parser.parse_args(argv)
-    current_names = terraform_state('list').decode().splitlines()
+    current_names = terraform_run('state', 'list')
```

Comment:
```suggestion
    current_names = terraform_run('state', 'list').splitlines()
```
Unless you changed `terraform_run` to always return a list.